### PR TITLE
LPS-156722 fix rest-buidler template erro

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:sf

File: modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/build.gradle [hunk 1]
```diff
@@ -27,6 +27,7 @@ dependencies {
 
 	restBuilder (group: "com.liferay", name: "com.liferay.portal.tools.rest.builder", version: "1.0.93") {force = true}
 #else
+ 	compileOnly group: "javax.annotation", name: "javax.annotation-api", version: "1.3.2"
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/build.gradle:30]: please move this below com.liferay.portal. dependencies are sorted alphabetically when possible

File: modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/pom.xml [hunk 2]
```diff
@@ -127,6 +127,12 @@
 #end
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>javax.annotation</groupId>
+			<artifactId>javax.annotation-api</artifactId>
+			<version>1.3.2</version>
+			<scope>provided</scope>
+		</dependency>
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/pom.xml:135]: the whitespace to the right of dependency isn't needed

**Review by petershin — CHANGES_REQUESTED** [review]

File: modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/pom.xml (hunk 2 above)
Comment (simonjhy) [reply modules/sdk/project-templates/project-templates-rest-builder/src/main/resources/archetype-resources/__artifactId__-impl/pom.xml:140]: done

**Comment by simonjhy** [comment]
ci:test:sf

**Comment by simonjhy** [comment]
ci:test:project-templates

**Comment by simonjhy** [comment]
ci:test:project-templates

**Pull request closed without merging**